client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -26,13 +26,16 @@ import { localize } from 'i18n-calypso';
 import { preventWidows } from 'lib/formatting';
 import { domainManagementTransferInPrecheck } from 'my-sites/domains/paths';
 import { recordStartTransferClickInThankYou } from 'state/domains/actions';
-import getCheckoutUpgradeIntent from 'state/selectors/get-checkout-upgrade-intent';
+import Gridicon from 'components/gridicon';
+import getCheckoutUpgradeIntent from '../../../state/selectors/get-checkout-upgrade-intent';
```

Comment:
Does this need to be a relative import?

`import getCheckoutUpgradeIntent from 'state/selectors/get-checkout-upgrade-intent';`